app/src/main/java/org/p2p/wallet/history/ui/history/adapter/holders/ProgressViewHolder.kt

```diff
@@ -0,0 +1,10 @@
+package org.p2p.wallet.history.ui.history.adapter.holders
+
+import android.view.ViewGroup
+import org.p2p.wallet.databinding.ItemProgressBinding
+import org.p2p.wallet.utils.viewbinding.inflateViewBinding
+
+class ProgressViewHolder(
```

Comment:
just moved all these holders to a package `holders`, it's a shame that git thinks it's a new file